src/i18n/en.ts

```diff
@@ -171,6 +171,7 @@ export const messages_en: IntlMessages = {
   'editor.do_not_stack_on_mobile.label': `Do Not Stack on Mobile`,
   'editor.end_time.label': `End Time`,
   'editor.fields.label': `Fields`,
+  'editor.font_color.label': `Font Color`,
```

Comment:
What do you think about using `editor.text_color.label`?

Reply:
me gustaba mas como quedaba color de fuente y despues tamaño de fuente. De lo contrario quedaria color de texto y despues tamaño de fuente. Igualmente estoy esperando los contenidos finales para hacer cualquier cambio

Reply:
Probablemente lo mejor sea usar simplemente "Color" (`editor.color.label`) como se usa en las tools existentes (por ejemplo heading):

![image](https://github.com/FromDoppler/unlayer-editor/assets/1157864/2072f81d-6df5-461d-9837-b8aa5fa1ebf6)
 
De esta forma vamos a ser consistentes con loo que ya existe y evitamos agregar una nueva variante.